### DISCUSSION TITLE
Short talk format and invitations

### DISCUSSION THREAD

**Discussion by ml-evs** [MADICES/MADICES-2022#17] [Ideas]
We need to start inviting initiatives to give talks in early January. We have a longlist in the README and should organize (probably between @kjappelbaum and I) who will invite who.

In order to send out invites, we need to agree on a format. 

I was imaging a kind of "show-and-tell" for 10 minutes, where each talk will briefly introduce their initiative, their approach to RDM (with technical details), and the issues/opportunities they face/foresee. Here is some text we could use for the invites (to be adjusted in this thread):

> We would like to invite you, or someone from <consortium> with your recommendation, to give a talk in the plenary sessions of the 2022 MADICES workshop on data interoperability in the chemical sciences. We would love to hear about <consortium>'s work in this area, down to the technical implementation details and future hopes and plans. 
>
> The talk should be 10-15 minutes, allowing 5-10 minutes for Q&A afterwards. These talks will ideally be delivered live (and recorded) on the first day of the workshop (a purely recorded talk is also fine) and used to motivate discussions in the breakout rooms.
>
> Our current draft ideas for breakout room discussions can be found on the [conference GitHub page](https://github.com/MADICES/MADICES-2022/discussions) - please feel free to contribute and provide feedback on there too!
> 
> As the new year is already well under way, we would appreciate it if you could give us a response by Friday 14th January. Please also spread the CECAM invitation amongst any colleagues that you think may be interested in participating in the discussion sessions - our aim is to write a perspective paper based on these discussions.

**Comment by kjappelbaum** [comment]
Thanks for bringing this up. I'd prefer if we could do this in 2021 (with perhaps a shorter deadline for coming back to us, e.g. Jan 10) as the deadline we currently have on the CECAM portal (and which is also reasonable) is something like Jan 22.

**Reply by ml-evs** [reply]
Fine by me! As I just said on Slack, I'm happy to start sending out these emails (maybe between Christmas and New Year)

**Comment by kjappelbaum** [comment]
@ml-evs  Regarding the timing: Do we want to have it all on the first day (which might be too much presentation) or a bunch of talks every day? 

We could aim for "themed" sessions: 

Monday: ELN developers
Tuesday: Industry
Wednesday: Initiatives 

However, for the discussions, it'd be good to have all the input so I'm a bit torn... 

If we agree on the timing, I'll start sending invites (I updated our list in the organizing team on GH)

**Reply by ml-evs** [reply]
Will we end up with enough speakers to have themed sessions? I was imagining the first few sessions just being introduced with 2-3 short talks more as "icebreakers". Not against the idea in principle, I think we can be quite flexible with the sessions/timetable.

I've also edited that list to mark who I will invite/which orgs should already be represented.

**Reply by sphuber** [reply]
I think the theme idea might make a lot of sense to help structure and guide discussions, especially if the breakout rooms that follow after map neatly onto it. Do agree that we would need sufficient speakers for these themes though.
